src/commands/pubsub-shardchannels.json

```diff
@@ -0,0 +1,23 @@
+{
+  "SHARDCHANNELS": {
+    "summary": "List active shard channels",
+    "complexity": "O(N) where N is the number of active shard channels, and assuming constant time pattern matching (relatively short channels and patterns)",
+    "group": "pubsub",
+    "since": "7.0.0",
+    "arity": -2,
+    "container": "PUBSUB",
+    "function": "pubsubCommand",
```

Comment:
the container and function seem wrong.

Reply:
I have introduced two subcommands under `PUBSUB` so I believe the function and container are correct. I think I should add the history to the command.

Reply:
ohh sorry. my bad.